src/common/icons/check-icon.tsx

```diff
@@ -6,6 +6,29 @@ import { NamedSFC } from '../react/named-sfc';
 
 export const CheckIcon = NamedSFC('CheckIcon', () => (
     <span className="check-container">
-        <span className="check-icon" />
+        <svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+            <svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <circle cx="8" cy="8" r="8" fill="#228722" />
```

Comment:
Does the fill color here need to change for high contrast mode?

Reply:
Good point!